RELEASE.md

```diff
@@ -0,0 +1,7 @@
+Release type: minor
+
+There was a bug with the JSON custom scalar example.
+
+This release fixes the example and adds JSON scalar type into `strawberry.scalars`.
+
+The same happens with `Base64`: We now have Base16, Base32 and Base64 into `strawberry.scalars`
```

Comment:
``````suggestion
This release adds the following scalar types:

- `JSON`
- `Base16`
- `Base32`
- `Base64`

they can be used like so:

```python
from strawberry.scalar import Base16, Base32, Base64, JSON

@strawberry.type
class Example:
    a: Base16
    b: Base32
    c: Base64
    d: JSON
```

``````